tests/Doctrine/Tests/DBAL/Types/TypeTest.php

```diff
@@ -28,7 +30,10 @@ public function defaultTypesProvider() : iterable
                 continue;
             }
 
-            yield [$constant->getValue()];
+            $constantValue = $constant->getValue();
+            assert(is_string($constantValue));
```

Comment:
Should we use PHPUnit assertions in tests? They are more specific and are always effective regardless of the runtime configuration.

Reply:
Sure